fe/estimator_abfe.py

```diff
@@ -338,14 +346,26 @@ def _deltaG(model, sys_params) -> Tuple[Tuple[float, List], np.array]:
         k_translation = 200.0
         k_rotation = 100.0
         start = time.time()
+        md_topo = mdtraj.Topology.from_openmm(model.openmm_topo)
+        with NamedTemporaryFile(suffix=".xtc") as temp:
+            with open(temp.name, "wb") as ofs:
+                ofs.write(results[-2].frames)
+            frames = mdtraj.load_xtc(temp.name, top=md_topo)
+            lhs_xs = frames.xyz
+        with NamedTemporaryFile(suffix=".xtc") as temp:
+            with open(temp.name, "wb") as ofs:
+                ofs.write(results[-1].frames)
+            frames = mdtraj.load_xtc(temp.name, top=md_topo)
+            rhs_xs = frames.xyz
+
         lhs_du, rhs_du, rotation_samples, translation_samples = endpoint_correction.estimate_delta_us(
             k_translation=k_translation,
             k_rotation=k_rotation,
             core_idxs=core_restr.get_idxs(),
             core_params=core_restr.params.reshape((-1,2)),
             beta=model.beta,
-            lhs_xs=results[-2].xs,
-            rhs_xs=results[-1].xs
+            lhs_xs=lhs_xs,
+            rhs_xs=rhs_xs
```

Comment:
Note that this is saved with 3 decimal places of precision. So if we're in units of nanometers, we can be precise up to 1/1000th of a nanometer, or 1/100th of an angstroms. Since the endpoint correction only looks at the energy differences in harmonic angles, forces, and orientational restraints, I think this is pretty safe.

I know that @maxentile  had concerns re: loss of precision in the nonbonded forces, but this doesn't matter for the endpoint correction.